qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDiagnostics.java

```diff
@@ -701,8 +702,14 @@ private ResolvedJavaTypeInfo validatePropertyPart(PropertyPart part, Section own
 			// ex : {@org.acme.Item item}
 			// "{item.XXXX}
 			Range range = QutePositionUtility.createRange(part);
+			String signature = baseType.getSignature();
+			String property = part.getPartName();
 			Diagnostic diagnostic = createDiagnostic(range, DiagnosticSeverity.Error, QuteErrorCode.UnknownProperty,
-					part.getPartName(), baseType.getSignature());
+					property, signature);
+			Map<String, String> data = new HashMap<>();

```

Comment:
I suggest that you do like Lemminx 

 * use JsonObject instead of map in order to support eclipse ide because iy works like this
 * * use an utility class to create the data that you need to store and get it